attack.go

```diff
@@ -25,6 +25,7 @@ func attackCmd() command {
 
 	fs.StringVar(&opts.name, "name", "", "Attack name")
 	fs.StringVar(&opts.targetsf, "targets", "stdin", "Targets file")
+	fs.StringVar(&opts.format, "format", "legacy", "Targets format [legacy, json]")
```

Comment:
Defaults could be constants.

Reply:
They could. I'll make this change together with the other constant related changes.